dashboards/traefik/traefik.json

```diff
@@ -108,7 +108,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\nlatest(traefik_entrypoint_open_connections) AS 'Open Connections'\nFACET instance, entrypoint\nWHERE instrumentation.name = 'remote-write'\nTIMESERIES\nLIMIT MAX"
+                "query": "FROM Metric SELECT\nlatest(traefik_entrypoint_open_connections) AS 'Open Connections'\nFACET instance, entrypoint\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} ) facet concat(prometheus_server,' - ',instance) \nTIMESERIES\nLIMIT MAX "
```

Comment:
2 `FACET` conditions in this query; recommend consolidating into:

```
FACET concat(prometheus_server,' - ',instance) AS 'Instance', entrypoint
```

Reply:
Thanks, changes made.